dwave/system/composites/embedding.py

```diff
@@ -289,7 +290,8 @@ def async_unembed(response):
             if return_embedding:
                 sampleset.info['embedding_context'].update(
                     embedding_parameters=embedding_parameters,
-                    chain_strength=embedding.chain_strength)
+                    chain_strength=embedding.chain_strength,
+                    break_points=break_points(response, embedding))
```

Comment:
This is a non-trivial performance hit. IMO we either should not do this by default or we need to write a more performant implementation of `break_points`.

Reply:
I agree, ideally this would be a lazy proxy.

But FWIW, `return_embedding` does default to `False`.

Reply:
That's true, unless for instance [the inspector is imported](https://github.com/dwavesystems/dwave-inspector/blob/6746435aa2c7c815e3dd6350770af697fcba97d8/dwave/inspector/adapters.py#L82).

My inclination is to not include this in the embedding composite for now, but document how to use the `TrackingComposite` to get the information in the docstring of `break_points()`.

Reply:
I updated the PR according to feedback except for this comment. Should I move the statistic to `TrackingComposite ` instead?

Reply:
I think the best thing is just to remove the statistic from the embedding composite altogether. I would then add an example to the `break_points` docstring showing how to calculate it, using the `TrackingComposite` to retrieve the relevant information.